MainRepository/Program.cs

```diff
@@ -1,7 +1,31 @@
-﻿internal class Program
+﻿//Armstrong number
+for (; ; )
 {
-    private static void Main(string[] args)
+    Console.Write("Enter the number or press [e] to exit: ");
+    string input = Console.ReadLine();
+    if (input.ToLower() == "e") break;
+    if (int.TryParse(input, out int n) && n > 0)
     {
-        Console.WriteLine("Hello, World!");
+        bool result = ArmStrong(n);
+        if (!result) Console.WriteLine($"The given {n} number is not a armstrong number");

```

Comment:
Usually we'll write the true condition first. So, 
if (result) Console.WriteLine ($"The given {n} number is armstrong number");
else Console.WriteLine ($"The given {n} number is not a armstrong number");